specification/monitor/resource-manager/Microsoft.Insights/stable/2021-05-01/metrics_API.json

```diff
@@ -0,0 +1,803 @@
+            "schema": {
+              "$ref": "#/definitions/MultiMetricResponse"
+            }
+          }
+        },
+        "x-ms-odata": "#/definitions/MetadataValue",
+        "x-ms-examples": {
+          "Get subscription level metric data": {
+            "$ref": "./examples/GetMultiResourceMetric.json"
+          },
+          "Post request for subscription level metric data": {
+            "$ref": "./examples/PostMultiResourceMetricBody.json"
+          },
+          "Get subscription level metric metadata": {
+            "$ref": "./examples/GetMultiResourceMetricMetadata.json"
+          }
+        }
+      }
+    },
+    "/{resourceUri}/providers/Microsoft.Insights/metrics": {
+      "get": {
+        "tags": [
+          "Metrics"
+        ],
+        "operationId": "Metrics_List",
+        "description": "**Lists the metric values for a resource**.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "#/parameters/TimespanParameter"
+          },
+          {
+            "$ref": "#/parameters/IntervalParameter"
+          },
+          {
+            "$ref": "#/parameters/MetricNamesParameter"
+          },
+          {
+            "$ref": "#/parameters/AggregationsParameter"
+          },
+          {
+            "$ref": "#/parameters/TopParameter"
+          },
+          {
+            "$ref": "#/parameters/OrderByParameter"
+          },
+          {
+            "$ref": "#/parameters/FilterParameter"
+          },
+          {
+            "$ref": "#/parameters/ResultTypeParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/MetricNamespaceParameter"
+          },
+          {
+            "$ref": "#/parameters/AutoAdjustTimegrain"
+          },
+          {
+            "$ref": "#/parameters/ValidateDimensions"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to get the list of metric values.",
+            "schema": {
+              "$ref": "#/definitions/Response"
+            }
+          }
+        },
+        "x-ms-odata": "#/definitions/MetadataValue",
+        "x-ms-examples": {
+          "Get Metric for data": {
+            "$ref": "./examples/GetMetric.json"
+          },
+          "Get Metric for metadata": {
+            "$ref": "./examples/GetMetricMetadata.json"
+          },
+          "Get Metric with error": {
+            "$ref": "./examples/GetMetricError.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "BodyParameters": {
```

Comment:
Maybe a more specific name (if List really uses a body). `BodyParameters` is not meaningful when this JSON is used together with 10+ other JSONs.